beacon-chain/sync/error.go

```diff
@@ -102,3 +105,8 @@ func readStatusCodeNoDeadline(stream network.Stream, encoding encoder.NetworkEnc
 
 	return b[0], string(msg.Message), nil
 }
+
+// only returns true for errors that are valid (no resets or expectedEOF errors).
+func isValidStreamError(err error) bool {
+	return err != nil && err.Error() != mux.ErrReset.Error() && err.Error() != helpers.ErrExpectedEOF.Error()
```

Comment:
```suggestion
	return err != nil && !errors.Is(err, mux.ErrReset) && !errors.Is(err, helpers.ErrExpectedEOF)
```
Wouldn't it be more clear to check using `errors.Is()` (available since Go 1.13)?